pkg/actors/actors.go

```diff
@@ -106,6 +109,7 @@ func NewActors(stateStore state.Store, appChannel channel.AppChannel, grpcConnec
 		evaluationLock:      &sync.RWMutex{},
 		evaluationBusy:      false,
 		evaluationChan:      make(chan bool),
+		appHealthy:          true,
```

Comment:
Should the initial value be false?

Reply:
No, when this code executed the Dapr runtime has already discovered that the application is started and responsive.